data/initrd/scripts/remote_log_setup

```diff
@@ -0,0 +1,66 @@
+#! /bin/sh
+#USAGE: ./script_name $remote_log_host_ip
+
+#preconditions
+echo -n "linking mkdir"
+if  ln -s /usr/bin/mkdir /bin/mkdir ; then
```

Comment:
This `if ... echo "ok" else echo "failed"` pattern is repeated several times, I think it could be shared in a function.